### PR TITLE
ignore warning created in figure-dev-deps

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

**Review by dstansby — CHANGES_REQUESTED** [review]
We should work out why this is warning, and try and fix it. As a minimum, ignoring this warning needs to have a link to a sunpy issue so we remember ignoring this doesn't get into a release.

**Comment by nabobalis** [comment]
It comes from this PR: https://github.com/astropy/astropy/pull/11260/files and astropy themselves ignore it.

**Comment by dstansby** [comment]
Thanks for the PR link - I think we should look into this further, as we don't want users being spammed with these warnings if they're spurious. I'll try and take a look in the next week or so.

**Comment by nabobalis** [comment]
True, this warning will most likely be emitted when we use the next version of astropy. 

I don't think that holding back on fixing the CI until we investigate this is worth it. We could open an issue to track this?

**Comment by dstansby** [comment]
Yeah, sorry if I wasn't clear earlier, I'm fine with opening an issue and merging this 👍

**Comment by Cadair** [comment]
This warning is most likely emitted from WCSLIB so I don't expect there to be a quick fix. Someone should track this down to a MWE, but in a new issue.

**Review by Cadair — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -171,6 +171,8 @@ filterwarnings =
     # an astropy table inside a test, which is generally only done when
     # debugging.
     ignore:the imp module is deprecated in favour of importlib:DeprecationWarning:ipykernel.iostream
+    # Recent change to astropy has made our figure tests emit this warning.
```
Comment (Cadair) [inline setup.cfg:174]:
```suggestion
    # Recent change to astropy has made our figure tests emit this warning.
    # https://github.com/sunpy/sunpy/issues/5073
```

**Pull request merged**